mne/preprocessing/ica.py

```diff
@@ -347,6 +348,14 @@ def fit(self, inst, picks=None, start=None, stop=None, decim=None,
         tstep : float
             Length of data chunks for artifact rejection in seconds.
             It only applies if `inst` is of type Raw.
+        reject_by_annotation : bool
+            Whether to omit bad segments from the data before fitting. If True,
+            annotated segments with a description that starts with 'bad' are
```

Comment:
Can you check to see if we're consistent with this -- i.e., it needs to start with `bad` as opposed to having `bad` in it anywhere, etc.?